crates/rbuilder/src/utils/error_storage.rs

```diff
@@ -34,18 +46,23 @@ pub async fn spawn_error_storage_writer(
     global_cancel: CancellationToken,
 ) -> eyre::Result<()> {
     let mut storage = ErrorEventStorage::new_from_path(db_path).await?;
```

Comment:
Should this return an Err at the beginning if there is no event queue?

Reply:
This is the func creating the queue!